src/products/components/ProductListTiles/ProductListTiles.tsx

```diff
@@ -42,6 +42,7 @@ export const ProductListTiles: React.FC<ProductListTilesProps> = ({
           gridTemplateColumns={{ mobile: 3, tablet: 5, desktop: 6 }}
           gap={9}
           padding={9}
+          __paddingTop={`calc(${vars.space[9]} - ${vars.space[5]}`}
```

Comment:
question: why not use space 4?
```suggestion
         paddingTop={4}

Reply:
@poulch space 4 is not equal to space `9-5`, as they follow a non-linear scale, it is a necessary offset calculation to make surrounding equal